dbms/src/Common/FailPoint.cpp

```diff
@@ -85,33 +85,42 @@ std::unordered_map<String, std::shared_ptr<FailPointChannel>> FailPointHelper::f
     M(force_remote_read_for_batch_cop)                       \
     M(force_context_path)                                    \
     M(force_slow_page_storage_snapshot_release)              \
-    M(force_change_all_blobs_to_read_only)
-
-#define APPLY_FOR_FAILPOINTS_ONCE_WITH_CHANNEL(M) \
-    M(pause_with_alter_locks_acquired)            \
-    M(hang_in_execution)                          \
-    M(pause_before_dt_background_delta_merge)     \
-    M(pause_until_dt_background_delta_merge)      \
-    M(pause_before_apply_raft_cmd)                \
-    M(pause_before_apply_raft_snapshot)           \
-    M(pause_until_apply_raft_snapshot)            \
+    M(force_change_all_blobs_to_read_only)                   \
+    M(unblock_query_init_after_write)
+
+
+#define APPLY_FOR_FAILPOINTS_ONCE_WITH_PAUSEABLE(M) \
```

Comment:
I think with_pauseable maybe a better name than with_channel. Do we need to make a change like that? @JaySon-Huang

Reply:
How about `APPLY_FOR_PAUSEABLE_FAILPOINTS`?

Reply:
Agree with `APPLY_FOR_PAUSEABLE_FAILPOINTS` and `APPLY_FOR_PAUSEABLE_FAILPOINTS_ONCE`